python/lsst/pipe/tasks/snapCombine.py

```diff
@@ -203,14 +202,7 @@ def run(self, snap0, snap1, defects=None):
 
         if self.config.doDiffIm:
             if self.config.doPsfMatch:
-                self.log.info("snapCombine psfMatch")
-                diffRet = self.diffim.run(snap0, snap1, "subtractExposures")
-                diffExp = diffRet.subtractedImage
-
-                # Measure centroid and width of kernel; dependent on ticket #1980
-                # Useful diagnostic for the degree of astrometric shift between snaps.
-                diffKern = diffRet.psfMatchingKernel
-                width, height = diffKern.getDimensions()
+                raise NotImplementedError("PSF-matching of snaps is not yet supported.")
```

Comment:
No, this functionality has never actually been supported. We will need to add it, but it will be cleaner to start fresh than to try to use this *very* old code.